priv/repo/seeds.exs

```diff
@@ -9,3 +9,384 @@
+      %{name: "Chicken"},
+      %{name: "Ground Beef"},
+      %{name: "Steak"},
+      %{name: "Pork"},
+      %{name: "Veggies"},
+      %{name: "Tofu"},
+      %{name: "No Protein"},
+      %{name: "Grilled Chicken"},
+      %{name: "Falafel"},
+      %{name: "Gyro"},
+      %{name: "Kofta (meatball)"},
+      %{name: "Shredded lemon chicken"},
+      %{name: "Mjedra (rice & lentils)"}
+    ]
+  },
+  %{
+    name: "Toppings",
+    option: [
+      %{name: "White rice"},
+      %{name: "Brown rice"},
+      %{name: "Cheese"},
+      %{name: "Black beans"},
+      %{name: "Pinto beans"},
+      %{name: "Lettuce"},
+      %{name: "Tomato"},
+      %{name: "Tom Salsa"},
+      %{name: "Corn salsa"},
+      %{name: "Guacamole"},
+      %{name: "Cilantro"},
+      %{name: "Sour cream"}
+    ]
+  },
+  %{name: "Spice Level", option: [%{name: "Regular"}, %{name: "Medium"}, %{name: "H-O-T!"}]},
+  %{
+    name: "Dressing",
+    option: [
+      %{name: "Tahani"},
+      %{name: "Balsamic Vinaigrette"},
+      %{name: "Honey Mustard"},
+      %{name: "Greek"},
+      %{name: "Italian"},
+      %{name: "Blue Cheese"},
+      %{name: "Caesar"}
+    ]
+  }
+]
+
+Enum.each(heading_options, fn hdg ->
+  heading = Lunchbot.Repo.get_by!(OptionHeadings, name: "#{hdg.name}")
+
+  Enum.each(hdg.option, fn option ->
+    Lunchbot.Repo.insert!(%Options{
+      name: option.name,
+      option_heading_id: heading.id
+    })
+  end)
+end)
+
+heading_items = [
+  %{
+    name: "Protein",
+    items: [
+      %{name: "Bowl"},
+      %{name: "Burrito"},
+      %{name: "GARDEN"},
+      %{name: "GREEK"},
+      %{name: "CAESAR"}
+    ]
+  },
+  %{name: "Toppings", items: [%{name: "Bowl"}, %{name: "Burrito"}]},
+  %{
+    name: "Spice Level",
+    items: [
+      %{name: "TOFU SAAG"},
+      %{name: "ALOO GOBHI"},
+      %{name: "CHANA MASALA"},
+      %{name: "BAINGAN BHARTA"},
+      %{name: "KATHAL & POTATO MASALA"},
+      %{name: "TOFU MANGO CURRY"},
+      %{name: "TOFU MASALA"},
+      %{name: "EGG & POTATO CURRY"},
+      %{name: "PANEER (cheese) & PEPPER MAKHANI"},
+      %{name: "MALAI KOFTA"},
+      %{name: "PANEER MUSHROOM KORMA"},
+      %{name: "MADRAS PANEER"},
+      %{name: "SAAG PANEER"},
+      %{name: "VEGETABLE TIKKA MASALA"},
+      %{name: "PANEER TIKKA MASALA"},
+      %{name: "CHICKPEA POUTINE"},
+      %{name: "CHICKEN TIKKA MASALA"},
+      %{name: "CHICKEN SAAG"},
+      %{name: "CHICKEN MUSHROOM KORMA"},
+      %{name: "CHICKEN & POTATO VINDALOO"},
+      %{name: "MANGO CHICKEN"},
+      %{name: "MADRAS CHICKEN"},
+      %{name: "CHICKEN KADHAI"},
+      %{name: "CHICKEN CURRY"},
+      %{name: "CHICKEN TIKKA POUTINE"}
+    ]
+  },
+  %{name: "Dressing", items: [%{name: "GARDEN"}, %{name: "GREEK"}, %{name: "CAESAR"}]}
+]
+
+Enum.each(heading_items, fn hdg ->
+  heading = Lunchbot.Repo.get_by!(OptionHeadings, name: "#{hdg.name}")
+
+  Enum.each(hdg.items, fn item ->
+    item = Lunchbot.Repo.get_by!(Item, name: "#{item.name}")
```

Comment:
Since there could be (and are) multiple records with the same name, use a query to get the appropriate records then iterate over each item record:

```
Enum.each(hdg.items, fn item ->
    items = from(i in Item, where: i.name == ^item.name) |> Lunchbot.Repo.all()
    Enum.each(items, fn it ->
      Lunchbot.Repo.insert!(%ItemOptionHeadings{
        item_id: it.id,
        option_heading_id: heading.id
      })
    end)
  end)
```

Then, import the Ecto.Query from/2 function at the top of your seeds file:
` import Ecto.Query, only: [from: 2]`